### ISSUE TITLE
Tweet Publish Date

### ISSUE THREAD

**Issue by CamronWalker** [DominikPieper/obsidian-ReadItLater#106]
### ⚠️ Please check that this feature request hasn't been suggested before.

- [X] I searched previous [Ideas in Discussions](https://github.com/DominikPieper/obsidian-ReadItLater/discussions/categories/ideas) didn't find any similar feature requests.
- [X] I searched previous [Issues](https://github.com/DominikPieper/obsidian-ReadItLater/labels/type%3A%20enhancement) didn't find any similar feature requests.

### 🔖 Feature description

Add the ability to insert the tweet's publish date. I'm proposing `%tweetPublishDate%`. This is for similar reasons to why I wanted `%videoPublishDate%` on YouTube https://github.com/DominikPieper/obsidian-ReadItLater/pull/105 but this time with `%tweetPublishDate%`. 

### ✔️ Solution

I know Twitter's embed API is limited but it does always include the publish date in the same spot of its html response (right before `</a></blockquote>`). This is how I got it working:
```js

.replace(/%tweetPublishDate%/g, this.parseTweetPublishDate(response.html));

...

parseTweetPublishDate(tContent: string){
  const splitContent = tContent.split("</a></blockquote>")
  const pubDate = splitContent[0].substring(splitContent[0].lastIndexOf('>') + 1)
  return moment(pubDate).format(this.settings.dateContentFmt)
}
```

I just want someone else's opinion. I'm pretty sure this will work consistently but there are some other ways I could get it. It even gets the right date when another tweet is quoted and there are multiple instances of `</a></blockquote>` in the embed html response.

### ❓ Alternatives

I know there are talks of using Nitter to get and download the images, but this is completely over my head https://github.com/DominikPieper/obsidian-ReadItLater/issues/91#issuecomment-1579247224.

**Comment by adamluckdev** [comment]
Added in #122

**Comment by adamluckdev** [comment]
Released in [0.4.0](https://github.com/DominikPieper/obsidian-ReadItLater/releases/tag/0.4.0)
